pkg/asset/tls/keypair.go

```diff
@@ -15,9 +15,9 @@ type KeyPairInterface interface {
 
 // KeyPair contains a private key and a public key.
 type KeyPair struct {
-	pvt   []byte
-	pub   []byte
-	files []*asset.File
+	Pvt      []byte
```

Comment:
do have strong opinion on it not being `Private` and `Public` ?

Reply:
I actually wanted to expose these fields, and they clash with the function names. No other reason, I strongly think that Pvt/Pub stinks as compared to Private/Public.

Reply:
hmm, lets keep it as is then. :/